module/preference/PreferenceDialog.kt

```diff
@@ -378,26 +406,23 @@ object PreferenceDialog {
         }
 
         class ProjectManager : PreferenceGroup(PROJECT_MANAGER) {
-            companion object {
-                private const val IGNORED_PATHS_PLACEHOLDER = ".git"
-            }
-
-            private val ignoredPathsString = preferenceMgr.ignoredPaths.joinToString(", ")
-            private var ignoredPaths = PreferenceField.TextInput(
+            private val ignoredPathsString = preferenceMgr.ignoredPaths.joinToString("\n")
+            private var ignoredPaths = PreferenceField.MultilineTextInput(
                 initValue = ignoredPathsString,
-                label = PROJECT_IGNORED_PATHS, placeholder = IGNORED_PATHS_PLACEHOLDER,
-                caption = IGNORED_PATHS_CAPTION
+                label = PROJECT_IGNORED_PATHS,
+                caption = IGNORED_PATHS_CAPTION,
+                fieldHeight = 5,
             )
 
             override val preferences: List<PreferenceField> = listOf(ignoredPaths)
 
             override fun submit() {
-                preferenceMgr.ignoredPaths = ignoredPaths.value.split(',').map { it.trim() }
+                preferenceMgr.ignoredPaths = ignoredPaths.value.text.split('\n').map { it.trim() }
                 ignoredPaths.modified = false
             }
 
             override fun reset() {
-                ignoredPaths.value = preferenceMgr.ignoredPaths.joinToString(", ")
+                ignoredPaths.value = preferenceMgr.ignoredPaths.joinToString("\n").let { TextFieldValue(it) }
```

Comment:
We now provide a new line delimited multiline text input field for ignored paths.